### PR TITLE
improve computed isGrid to native function

### PR DESCRIPTION
**Pull request by AmilKey**

### CODE REVIEW

File: addon/list-view-mixin.js [hunk 1]
```diff
@@ -99,9 +99,7 @@ export default Ember.Mixin.create({
   paddingCount: 1,
   _cachedPos: 0,
 
-  _isGrid: Ember.computed('columnCount', function() {
-    return this.get('columnCount') > 1;
-  }).readOnly(),
+  _isGrid: Ember.computed.gt('columnCount', 1),
```
Comment (stefanpenner) [inline addon/list-view-mixin.js]: still needs `readOnly`

**Comment by stefanpenner** [comment]
left a comment but still +1

**Comment by AmilKey** [comment]
@stefanpenner updated, added readOnly

File: addon/list-view-mixin.js (hunk 1 above)
Comment (AmilKey) [inline addon/list-view-mixin.js]: @stefanpenner updated

**Comment by jonnii** [comment]
@stefanpenner totally off topic, but I always assumed that `Ember.computed.*` properties were `readOnly`. Other than causing an error if you try to set it, are there any other advantages to marking a property as `readOnly`?

**Pull request merged**